src/hooks/useClickOutsideHandler.ts

```
@@ -0,0 +1,25 @@
+import { useEffect } from 'react';
+
+function useClickOutsideHandler(
+  targetRef: React.RefObject<HTMLElement | null>,
+  executionFunction: Function,
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)